packages/cyberstorm/src/components/Toast/Toast.module.css

```diff
@@ -0,0 +1,181 @@
+.root {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 12px 24px 5px rgb(0 0 0 / 0.65);
+}
+
+.contentWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-left: 3px solid var(--feature-color);
+  border-radius: var(--border-radius--4);
+  background-color: var(--background-color);
+
+  --background-color: var(--alert-info-bg-color);
+  --feature-color: var(--alert-info-accent-color);
+  --text-color: var(--alert-info-text-color);
+}
+
+.content {
+  display: flex;
+  gap: var(--space--16);
+  align-items: flex-start;
+  align-self: stretch;
+  width: 100%;
+  padding: 1rem 3.25rem calc(1rem - 0.125rem) 1.5rem;
+}
+
+.closeIconWrapper {
+  position: absolute;
+  top: 0.5rem;
+  right: 0.5rem;
+  background-color: transparent;
+}
+
+.closeIcon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 1em;
+  height: 1em;
+  color: var(--color-text--tertiary);
+}
+
+.toastProgress {
+  width: 100%;
+  height: 0.125rem;
+  background-color: rgb(0 0 0 / 0.3);
+}
+
+.toastProgressBar {
+  height: 100%;
+  background-color: var(--feature-color);
+  animation: progress-bar var(--bar-timer) linear forwards;
+}
+
+/* Styles for Toast's text */
+.message {
+  display: flex;
+  flex: 1 0 0;
+  min-height: calc(1rem * var(--line-height--m));
+  color: var(--text-color);
```

Comment:
Not the expert on this either, but the color and the three rows below it seem to use what I would assume are the standard values? Do we need to explicitly define them here? Or is it our "best practice" to always define them, even if they aren't actually needed?

Reply:
AFAIK we don't have any agreed best practices. I've planned these to be established after the MVP, while we work on the style library.

BUT, I've tried to define anything relating to fonts/text on the nearest parent element to the text, just as a precautions, to keep everything in check if some global variables, etc, go missing.

I'd say let's go forward with this for now on and go over this, when the style library is being worked on.

Reply:
TS-1957